src/main/webapp/app/exercises/programming/participate/programming-submission.service.ts

```diff
@@ -570,16 +570,22 @@ export class ProgrammingSubmissionService implements IProgrammingSubmissionServi
         if (lock) {
             params = params.set('lock', 'true');
         }
-
-        return this.http.get<ProgrammingSubmission>(url, { params });
+        const returnValue = this.http.get<ProgrammingSubmission>(url, { params });
+        return returnValue;
```

Comment:
I think the additional variable returnValue is not needed here.

Reply:
yes, I will remove that 👍

Reply:
I removed it :)